### PR TITLE
Don't autofocus fields in IFDB forms

### LINKED ISSUE #397: Autofocus on title field when editing listings
When /editgame loads, it automatically jumps your cursor to the **Title** field:
The top of an Edit Game Listing page, with the Title field highlighted

This makes sense when creating a new listing. But when editing, it just forces me to wait an extra couple seconds and then click out so I don't accidentally overwrite the title.

The same thing happens on /editcomp, /editlist, /editclub. /editprofile similarly autofocuses the **Email Address** field.

### PR DESCRIPTION
**Pull request by dfabulich**
In many pages on the site, we try to guess which field you want to edit most, and autofocus it.

I find it mildly annoying on desktop; it's super obnoxious on mobile, because it can cause the screen to scroll and the keyboard to popup, obscuring half the screen.

Now, we don't do that.

Fixes #397

### CODE REVIEW

**Comment by dfabulich** [comment]
@salty-horse Do you want to review this? (I could just merge it…?)

File: www/lostact [hunk 1]
```diff
@@ -26,7 +26,12 @@ if (!$showForm)
 //
 // Main form display
 //
+<<<<<<< main
```
Comment (salty-horse) [inline www/lostact:29]: merge conflict tags

File: www/lostpass [hunk 2]
```diff
@@ -26,7 +26,12 @@ if (!$showForm)
 //
 // Main form display
 //
+<<<<<<< main
```
Comment (salty-horse) [inline www/lostpass:29]: merge conflict tags

File: www/newuser [hunk 3]
```diff
@@ -28,8 +28,13 @@ if (!$showForm)
 // Main form display
 //
 
+<<<<<<< main
```
Comment (salty-horse) [inline www/newuser:31]: merge conflict tags

**Comment by dfabulich** [comment]
how embarrassing!

**Review by salty-horse — APPROVED** [review]

**Pull request merged**